pkgs/shells/zsh/default.nix

```diff
@@ -18,6 +18,12 @@ stdenv.mkDerivation {
     sha256 = "1s3yww0mzgvpc48kp0x868mm3gbna42sbgzya0nknj0x5hn2jq3j";
   };
 
+  patches = [
+    # Bracketed-paste-magic error in Putty due to empty $PASTED variable
+    # http://www.zsh.org/mla/workers/2019/msg00808.html
```

Comment:
If this patch is included in the next version of zsh, please append this to this comment.